specification/azsadmin/resource-manager/deployment/Microsoft.Deployment.Admin/preview/2019-01-01/ActionPlanOperation.json

```diff
@@ -45,6 +45,15 @@
             "schema": {
               "$ref": "#/definitions/ActionPlanOperationsList"
             }
+          },
+          "404": {
```

Comment:
404 error response needs to be removed everywhere, having default error response is enough